src/Slices/Notification/UI/Drawer/Item.tsx

```diff
@@ -84,7 +84,7 @@ const ActionList: React.FC<Props> = ({ notification, onUpdate }) => {
         <DropdownItem
           key="cleared"
           component="button"
-          onClick={() => onUpdate({ read: true, cleared: true })}
+          onClick={() => onUpdate({ cleared: true })}
```

Comment:
Question, there's a reason why they are set on read, isn't this supposed to trigger an API call ? When you go to the notification page, you can filter the notifications based on their (un)read status. I rather think the issue lies in the updateMethod instead of the passed arguments.

Reply:
I checked once again and notification Item has a onclick and event was propagated to it which resulted of doubling request where latter one has body to only mark notification as "read". So my attempt resolved issue but in sub-optimal way, I'll change that :)